packages/web-app-files/src/components/SideBar/Shares/Collaborators/NewCollaborator.vue

```diff
@@ -1,10 +1,11 @@
 <template>
-  <div class="files-collaborators-collaborator-add-dialog">
+  <div class="files-collaborators-collaborator-add-dialog" data-testid="new-collaborator">
     <div class="oc-mb">
       <oc-select
         id="files-share-invite-input"
         ref="ocSharingAutocomplete"
         v-model="selectedCollaborators"
+        data-testid="new-collaborator-select"
```

Comment:
Theoretically could be removed in favor of the already exisiting `id` on that button